app/steps/ui/will/neworiginal/index.js

```diff
@@ -0,0 +1,22 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/original.json');
+
+module.exports = class NewWillOriginal extends ValidationStep {
+
+    static getUrl() {
+        return '/new-will-original';
+    }
+
+    nextStepUrl(ctx) {
+        return this.next(ctx).constructor.getUrl('notOriginal');
+    }
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'original', value: json.optionYes, choice: 'isOriginal'}
+            ]
+        };
+        return nextStepOptions;
+    }
+};
```

Comment:
Please put 'use strict'; on the first line of this file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#strict-mode for an example